mod-linked-data/src/main/resources/citation/mod-linked-data/features/suppress-flags/samples/create-work-request.json

```diff
@@ -0,0 +1,145 @@
+{
+  "resource":{
+    "http://bibfra.me/vocab/lite/Work":{
+      "http://bibfra.me/vocab/marc/title":[
+        {
+          "http://bibfra.me/vocab/marc/Title":{
+            "http://bibfra.me/vocab/bflc/nonSortNum":[
+              "4"
+            ],
+            "http://bibfra.me/vocab/marc/mainTitle":[
+              "suppress_flags"
+            ],
+            "http://bibfra.me/vocab/marc/partNumber":[
+              "1"
+            ],
+            "http://bibfra.me/vocab/marc/partName":[
+              "part 1"
+            ]
+          }
+        }
+      ],
+      "http://bibfra.me/vocab/marc/governmentPublication":[
```

Comment:
Minor - You can simplify this request by removing everything other than "http://bibfra.me/vocab/marc/title" from the request.

Reply:
already tried to do it, but test got failed:
<img width="350" alt="image" src="https://github.com/user-attachments/assets/a3b7512f-325d-4f15-922b-f762144940a7">
so, decided to keep it as is

Reply:
Following requests worked for me

{
  "resource":{
    "http://bibfra.me/vocab/lite/Work":{
      "http://bibfra.me/vocab/marc/title":[
        {
          "http://bibfra.me/vocab/marc/Title":{
            "http://bibfra.me/vocab/bflc/nonSortNum":[
              "4"
            ],
            "http://bibfra.me/vocab/marc/mainTitle":[
              "suppress_flags"
            ],
            "http://bibfra.me/vocab/marc/partNumber":[
              "1"
            ],
            "http://bibfra.me/vocab/marc/partName":[
              "part 1"
            ]
          }
        }
      ],
      "http://bibfra.me/vocab/marc/content":[
        {
          "http://bibfra.me/vocab/marc/term":[
            "cartographic image"
          ],
          "http://bibfra.me/vocab/lite/link":[
            "http://id.loc.gov/vocabulary/contentTypes/cri"
          ]
        }
      ]
    }
  }
}

Reply:
thank you PK, have updated pr